ui/library/src/utils/MaxVisibleElement.ts

```diff
@@ -0,0 +1,17 @@
+import { Nullable } from '../components/types/utils';
+
+type VisibleElementKey = Nullable<number | string>;
+
+export function getMaxVisibleElement(
+  visibleElements: Map<VisibleElementKey, number>
+): VisibleElementKey {
```

Comment:
It's weird to me seeing a function called `getMaxVisibleElement` but actually it returns `VisibleElementKey`. If you are not intending to change the function name, maybe changing `VisibleElementKey` to `VisibleElement` will be better.